.github/workflows/debug-partner-image.yaml

```diff
@@ -66,6 +66,22 @@ jobs:
           username: ${{ secrets.QUAY_ROBOT_USERNAME }}
           password: ${{ secrets.QUAY_ROBOT_TOKEN }}
 
+      - name: Add metadata
+        uses: docker/metadata-action@v4
+        id: meta
+        with:
+          images: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}
```

Comment:
Does this also have to have the `:tag` as well?

Reply:
I don't think so, I didn't try this action with quay.io, only with ghcr.io.